scanner/src/main/kotlin/Scanner.kt

```diff
@@ -156,7 +156,7 @@ class Scanner(
         return ortResult.copy(scanner = scannerRun)
     }
 
-    suspend fun scan(packages: Set<Package>, context: ScanContext): ScannerRun {
+    internal suspend fun scan(packages: Set<Package>, context: ScanContext): ScannerRun {
```

Comment:
@mnonnenmacher is this maybe used by ORT Server?

Reply:
No, it doesn't. But I think for the API it would be nice to offer a function that does not require a complete ORT result object.

Reply:
Fine with me, but in general I think we should start collecting changes like this because we too often make major releases where the changes actually affect probably no one, like here.

Reply:
Something like that, the function has been in the code base for two years so it would not hurt to keep the change on a branch until the next meaningful breaking change is merged. But it's a separate discussion we should not have in this PR.